test/nonnegative_bridge.jl

```diff
@@ -40,14 +45,18 @@ function MOI.supports_constraint(::Type{NonnegativeBridge{T}},
                                  ::Type{Nonnegative}) where T
     return true
 end
+
+function MOIB.added_constrained_variable_types(::Type{NonnegativeBridge{T, F}}) where {T, F}
```

Comment:
You can just do `Type{<:NonnegativeBridge}` as you don't use `T` and `F` inside the function